src/pages/MapPage/components/Filter/FilterDrawer.js

```diff
@@ -144,34 +139,34 @@ const FilterDrawer = (props) => {
               <Grid container spacing={2}>
                 {target.map((discovery) => (
                   <Grid key={discovery.targetName} item xs={4}>
-                    <Button
-                      variant='contained'
+                    <CustomButton
                       fullWidth
-                      size='small'
                       color={
                         currentTarget !== discovery.targetName
                           ? 'default'
                           : 'primary'
                       }
+                      variant='contained'
+                      size='small'
                       onClick={() => changeCurrentTarget(discovery.targetName)}
-                    >
-                      {discovery.targetName}
-                    </Button>
+                      children={discovery.targetName}
+                    ></CustomButton>
                   </Grid>
                 ))}
               </Grid>
             </Grid>
           </Grid>
         </Box>
-        <Button
+
+        <CustomButton
           className={classes.button}
-          onClick={handleFinishFilter}
           color={final ? 'primary' : 'default'}
-          disabled={!final}
           variant='contained'
-        >
-          加入
-        </Button>
+          onClick={handleFinishFilter}
+          disabled={!final}
+          children='加入'
+          borderRadius='20px'
```

Comment:
border-radius should be write in the CustomButton file. If we have two version of border-radius, create a prop to choose which to use, we should not passing the style value here.